server/server.js

```diff
@@ -37,6 +37,18 @@ app.get('/api/artist/:name', (req, res) => {
   })
 })
 
+app.get('/api/artistinfo/:id', (req, res) => {
```

Comment:
Hmm po co Ci ten `route`? Chyba nigdzie z niego nie korzystasz.

Reply:
Najpierw planowalem wyciagac dane, ale uznalem ze skoro od razu i tak wyciagam jakies dane przy tworzeniu listu, to nie ma sensu tego powtoarzac, tylko tam wyciagne ich wiecje po prostu. Z tego zrobie wyciaganie relatedArtists i albumow artysty, ale w sumie moge wywalic tymczasowo albo chociaz zakomentowac ;p